src/Plugin/Block/ActivityFinder4Block.php

```diff
@@ -146,6 +167,12 @@ public function build() {
 
     $sort_options = $backend->getSortOptions();
 
+    $bs_version = 4;
```

Comment:
BS 4 by default... sounds reasonable; but we initially developed it for the BS3. No need to change anything... just keep this in mind ;)